.github/workflows/build.yml

```diff
@@ -29,6 +29,10 @@ jobs:
                         php: 8.0
                         symfony: ^5.4
                         twig: ^3.0
+                    -
+                        php: 8.1
+                        symfony: ^5.4
```

Comment:
Why didn't you add a build for Symfony 4.4 and PHP 8.1?

Reply:
done